dotenv/src/lib.rs

```diff
@@ -36,7 +36,7 @@ static START: Once = Once::new();
 /// ```
 pub fn var<K: AsRef<OsStr>>(key: K) -> Result<String> {
     START.call_once(|| {
-        dotenv().ok();
+        dotenv();
```

Comment:
This is why `.ok()` has been used, it silences the error without panicking the program. The intention of the `var` function is to only load a .env file _if it exists_. As you can see, Clippy has some issues with just leaving the error unhandled. 

Probably best to keep it as it was.

Reply:
I agree. The `.ok()` serves a purpose here in ignoring the error, which is the intended behaviour.

If you prefer, you can also do this:
```rust
let _ = dotenvy::dotenv();  // another common way to ignore an error
```

Reply:
reverted this change